### DISCUSSION TITLE
ROCm 5.7.0 Release

### DISCUSSION THREAD

**Discussion by saadrahim** [ROCm/ROCm#2474] [Announcements]
ROCm 5.7.0 includes many new features. Please see the complete [release notes](https://rocm.docs.amd.com/en/docs-5.7.0/release.html) New features include: a new library (hipTensor), debugger (ROCgdb) support for Fortran and OMPD, and optimizations for rocRAND and MIVisionX. Address sanitizer for host and device code (GPU) is now available as a beta. Note that ROCm 5.7.0 is EOS for MI50. 5.7 versions of ROCm are the last major release in the ROCm 5 series. This release is Linux-only. 

Important: The next major ROCm release (ROCm 6.0) will not be backward compatible with the ROCm 5 series. Changes will include: splitting LLVM packages into more manageable sizes, changes to the HIP runtime API that is not backward compatible, splitting rocRAND and hipRAND into separate packages, and reorganizing our file structure.

<hr /><em>This discussion was created from the release <a href='https://github.com/RadeonOpenCompute/ROCm/releases/tag/'>ROCm 5.7.0 Release</a>.</em>

**Comment by johnnynunez** [comment]
When rdna3 official support? Rocm6.0?

**Comment by sofiageo** [comment]
hipTensor is missing from the Ubuntu release. Another minor thing I noticed is hsa-amd-aqlprofile filename duplicates the rocm version (hsa-amd-aqlprofile_1.0.0.**50700**.**50700**-63~22.04_amd64.deb) but i don't know if it's worth reporting.

**Reply by cgmillette** [reply]
Hi there @sofiageo,

hipTensor packages are currently available in the 5.7 repos:

http://repo.radeon.com/rocm/apt/debian/pool/main/h/
http://repo.radeon.com/rocm/yum/5.7/main/

Could you help clarify your experience with missing hipTensor libraries? Would be happy to help.

**Reply by sofiageo** [reply]
Sorry, I meant that I believe that hipTensor is not included in the amdgpu-install script. I'm not using the Ubuntu packages myself, I just notice these things when "repackaging" for Arch Linux.
